src/main/kotlin/org/codefreak/codefreak/service/AnswerService.kt

```diff
@@ -133,5 +133,19 @@ class AnswerService : BaseService() {
   }
 
   private fun Task.isTesting() =
-      this.owner == FrontendUtil.getCurrentUser() || FrontendUtil.getCurrentUser().hasAuthority(Authority.ROLE_ADMIN)
+    this.owner == FrontendUtil.getCurrentUser() || FrontendUtil.getCurrentUser().hasAuthority(Authority.ROLE_ADMIN)
+
+  fun saveCommitId(answerId: UUID, commitID: String): Answer {
+    val answer = answerRepository.findById(answerId).get()
+    answer.commitId = commitID
+    return answerRepository.save(answer)
+  }
+
+  fun getCommitId(answerId: UUID): String {
+    try {
+      return answerRepository.findById(answerId).get().commitId
+    } catch (err: Exception) {
+      return ""
+    }
```

Comment:
```kotlin
return try {
  answerRepository.findById(answerId).get().commitId
} catch (err: NoSuchElementException) {
  ""
}
```

The exception you need to catch is a `java.util.NoSuchElementException`. Otherwise every other unrelated exception would be caught here too.